### DISCUSSION TITLE
Consider MicroPython support

### DISCUSSION THREAD

**Discussion by DragoonAethis** [Kozea/Radicale#1395] [Ideas]
MicroPython / https://github.com/micropython / is an extremely lightweight implementation of Python 3.4 (with some 3.5 elements too) aimed at embedded devices - recently it became available in OpenWRT repositories. Regular Python implementation is very heavy (even in its "light" variant, as available in the OpenWRT repo) and MicroPython support might allow using Radicale on low-end routers where there's not enough memory to deal with the full Python implementation (or mid-end routers where its manufacturer graced everyone with 16MB of flash ;-;).

**Comment by liZe** [comment]
That's a good idea! It's probably easier to fork the 1.1.x branch now that Radicale depends on vobject.
